backend/build.gradle

```
@@ -41,6 +44,8 @@ dependencies {
 	implementation 'io.jsonwebtoken:jjwt-api:0.11.5'
 	runtimeOnly 'io.jsonwebtoken:jjwt-impl:0.11.5'
 	runtimeOnly 'io.jsonwebtoken:jjwt-jackson:0.11.5'
+	implementation 'org.springframework.boot:spring-boot-starter-log4j2'
+	implementation group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-yaml', version: '2.10.3'
```

group, name, version 명시를 따로 안해줘도 될 것 같아요
```suggestion
	implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
```